src/main/java/com/amazon/ion/impl/_private/SuppressFBWarnings.java

```diff
@@ -0,0 +1,13 @@
+package com.amazon.ion.impl._private;
+
+/**
+ * SpotBugs looks for an annotation called `SuppressFBWarnings` regardless of the package it is in.
+ * This will allow us to disable some SpotBugs rules when we want e.g. to allow switch case fallthrough.
+ */
```

Comment:
Is this preferable to having a compile-time dependency on `com.github.spotbugs:spotbugs-annotations`?

Reply:
Yes, this is preferable.

If we introduce spotbugs annotations as a `implementation` or `runtimeOnly` dependency, either it's a transitive dependency for consumers or it needs to be bundled into the superjar. If it's bundled into the superjar, then we  are distributing it and we would need to figure out the implications of distributing software that is licensed as LGPL.

We cannot introduce spotbugs annotations as a `compileOnly` dependency, because it will cause there to be an unresolved package when we try to run the tests, or when anyone else tries to use `ion-java` without having spotbugs-annotations in their classpath.

Reply:
Does Spotbugs not run locally? Why did we just start seeing these now?